vaadin-spreadsheet-testbench-api/pom.xml

```diff
@@ -40,13 +46,13 @@
         <dependency>
             <groupId>com.vaadin</groupId>
             <artifactId>vaadin-testbench-api</artifactId>
-            <version>8.0.0</version>
+            <version>8.14.3</version>
             <scope>provided</scope>
         </dependency>
         <dependency>
             <groupId>com.vaadin</groupId>
             <artifactId>vaadin-testbench-core</artifactId>
-            <version>5.1.0</version>
+            <version>5.2.0</version>
```

Comment:
5.3.0 exists

Reply:
My thought was to keep the version in sync with the core, but I suppose we can update the version there as well. I'll bump the number in a moment.